src/js/constants/ModalKeys.js

```diff
@@ -0,0 +1,6 @@
+let MODAL_KEYS = {
+  SERVICE_FORM_MODAL: 'SERVICE_FORM_MODAL',
+  SERVICE_GROUP_FORM_MODAL: 'SERVICE_GROUP_FORM_MODAL'
+};
+
+module.exports = MODAL_KEYS;
```

Comment:
Actually that's a bad idea, I can see how we can expand these with other modal ids in the future. nvm